acquia_lift.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Installation file for Acquia Lift module.
+ */
+
+use Drupal\acquia_lift\Service\Helper\SettingsHelper;
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Remove cache directory if module is uninstalled.
+ */
+function acquia_lift_uninstall() {
+  // Purging all Acquia Lift settings and js libraries.
+  drupal_flush_all_caches();
```

Comment:
It seems like this should be unnecessary as the asset caches and config caches are cleared during the module uninstall process.  https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21ModuleInstaller.php/function/ModuleInstaller%3A%3Auninstall/8